test/e2e/crcsuite/crcsuite.go

```diff
@@ -375,12 +377,27 @@ func StartCRCWithDefaultBundleSucceedsOrFails(expected string) error {
 	return err
 }
 
+func RunCRCSetupSucceedsOrFails(expected string) error {
```

Comment:
Wouldn't it be simpler to, instead of this new function and a pattern matching cucumber expression, just insert something like the following 
```go
if !bundleDownloadedAutomatic && (command == "setup" || command == "start") {
	command = fmt.Sprintf("%s -b %s", command, bundleLocation)
}
```
above this line: https://github.com/code-ready/crc/blob/ff5d17656d1677301b42b78297e18e98f4a4618a/test/e2e/crcsuite/crcsuite.go#L427
Then we'd require only a handful of 1-word changes in the feature files.

Reply:
@jsliacan I did try that but something was not working locally so I went with this path but I did try again and it is working.